### PR TITLE
1429: Adding Bulk Add to the Parts Marketplace

### LINKED ISSUE #1429: Enhancement to Purchase Parts menu option
### Environment
Using Snapshot 47.3 

### Description
Under the Overview tab, Parts in use, there's an option while GM mode is engaged to add parts in bulk.  Under the Marketplace, Purchase Parts, while in GM mode you can only add one part at a time.  Would it be possible to *add* add parts in bulk to that section?

### PR DESCRIPTION
**Pull request by Windchild292**
This fixes #1429. It also reorders the buttons to be consistent with them elsewhere in the program and hides GM buttons when not in GM mode.

### CODE REVIEW

File: MekHQ/src/mekhq/gui/dialog/PopupValueChoiceDialog.java [hunk 1]
```diff
@@ -1,28 +1,31 @@
 /*
- * AlertPopup.java
+ * Copyright (c) 2013 The MegaMek Team. All rights reserved.
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/gui/dialog/PopupValueChoiceDialog.java]:
```suggestion
 * Copyright (c) 2010, 2013 The MegaMek Team. All rights reserved.
```
Per:
> `* Created on Jan 6, 2010, 10:46:02 PM`

Comment (Windchild292) [reply MekHQ/src/mekhq/gui/dialog/PopupValueChoiceDialog.java]: The file is actually 2013, the header was copied and never updated with the date

Comment (sixlettervariables) [reply MekHQ/src/mekhq/gui/dialog/PopupValueChoiceDialog.java]: As in the header was copied from another file?

Comment (Windchild292) [reply MekHQ/src/mekhq/gui/dialog/PopupValueChoiceDialog.java]: Yep

**Pull request merged**